hazelcast/src/main/java/com/hazelcast/internal/util/LatencyDistribution.java

```diff
@@ -65,15 +107,15 @@ public long bucket(int bucket) {
     /**
      * The maximum value that can be placed in a bucket.
      */
-    public static long bucketMaxUs(int bucket) {
-        return bucket == 0 ? 0 : (1L << bucket) - 1;
+    static int bucketMaxUs(int bucket) {
```

Comment:
Another indication that we are losing the last bucket.

2^32=2B. Since we measure in us, we go from a maximum latency that can be measured from 60m to 30m.

Reply:
No, what can be measured as max are same with the current code base, both are 2^31.